content/departments/cloud/technical-docs/v1.1/mi1-1_upgrade_process.md

```diff
@@ -71,35 +71,53 @@ export VERSION=3.40.0
 
 #### Automated Upgrade (recommended)
 
+Using `mi` you can generate commands to trigger automated upgrades for instances that are not yet upgraded:
+
 ```sh
-gh workflow run mi_upgrade.yml -f customer=$CUSTOMER -f version=$VERSION
+# For trial instances
+mi workflow upgrade -target $VERSION -instance-type=trial
+# For customer instances
+mi workflow upgrade -target $VERSION -instance-type=production
```

Comment:
I think we want to limit the number of concurrent upgrades to limit the blast radius

Reply:
> I think we want to limit the number of concurrent upgrades to limit the blast radius

This recommendation is a few lines beneath this text :)

> or output a list of commands where an operator to execute manually

That's what `mi workflow upgrade` does today - the output must be copy-pasted to be executed

Reply:
oops, I should read more careful